pages/docs/advanced/meta.en-US.json

```diff
@@ -1,5 +1,5 @@
 {
-  "return-values": "Return Values",
+  "understanding-swr": "Understanding SWR",
```

Comment:
```suggestion
  "understanding": "Understanding SWR",
```

Reply:
@huozhi Thank you! I've applied your suggestion and update all links and file names 👍